lib/msf/core/exploit/http/joomla/version.rb

```diff
@@ -0,0 +1,95 @@
+# -*- coding: binary -*-
+
+module Msf::Exploit::Remote::HTTP::Joomla::Version
+
+  # Returns the Joomla version.
+  #
+  # @return [String] Joomla version (if found)
+  # @return [NilClass] No Joomla version found.
+  def joomla_version
+    files = [
+      'language/en-GB/en-GB.xml',
```

Comment:
we should add `administrator/manifests/files/joomla.xml` to this list. The regexes below should already be able to parse this file.